nengo_gui/static/components/netgraph_item.js

```diff
@@ -353,14 +358,14 @@ Nengo.NetGraphItem.prototype.generate_menu = function () {
             items.push(['XY-value', function() {self.create_graph('XYValue');}]);
         }
         if (this.html_node) {
-            items.push(['HTML', function() {self.create_graph('HTMLView');}])
+            items.push(['HTML', function() {self.create_graph('HTMLView');}]);
```

Comment:
These changes and the ones below it are not part of this PR; they're part of #566